atd-vze/src/views/Crashes/Maps/customGeocoderMapController.js

```diff
@@ -0,0 +1,46 @@
+import { MapController } from "react-map-gl";
+
+export class CustomGeocoderMapController extends MapController {
+  _onPan(event) {
```

Comment:
so these methods overwrite the same `MapController` methods, I'm assuming?

Reply:
now digging into your links, I'm seeing that's exactly what it does. Can you add a comment to this file that gives a brief explanation and links to [the examples](https://github.com/uber/react-map-gl/blob/master/docs/advanced/custom-map-controller.md)?

Reply:
Great idea - I added in some documentation so it is easier to see what is going on. I'm sure that we will see this come up again with this library too.